docs/source/serving/sky-serve.rst

```diff
@@ -490,7 +490,7 @@ You can see the controller with :code:`sky status` and refresh its status by usi
 
 .. _customizing-sky-serve-controller-resources:
 
-Customizing SkyServe controller resources
+Customizing SkyServe Controller Resources
```

Comment:
after #4664  we can leave this in sentence case
https://github.com/skypilot-org/skypilot/blob/0a077c48616f853eee9e598f353384dfee2eb689/docs/README.md?plain=1#L4-L7